dobromir-hristov/swift-docc-render
# Add sidenav hero

Bug/issue #, if applicable: 89092590

## Summary

Implements a Hero section into Docc-render, that is to be used in tandem with the new Sidebar functionality

![image](https://user-images.githubusercontent.com/9863944/154540539-f510e75d-8991-45e2-8bc7-af63c152f27f.png)


## Dependencies

#51 

## Testing

Make sure you download and unzip the `.doccarchive` - [SlothCreator.doccarchive.zip](https://github.com/apple/swift-docc-render/files/8082216/SlothCreator.doccarchive.1.zip)


Steps:
1. run `VUE_APP_DEV_SERVER_PROXY=/path/to/SlothCreator.doccarchive npm run serve`
2. Visit http://localhost:8080/documentation/slothcreator
3. Assert there is a hero section with icons of the same type, as the items in the sidebar.

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
